### PR TITLE
Slides for #72 off by one++

### PR DESCRIPTION
**Pull request by MarlonSchultz**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 _site/
 .jekyll-metadata
 node_modules
+.idea/
```
Comment (lucasrangit) [inline .gitignore]: What's this?

**Comment by lucasrangit** [comment]
@MarlonSchultz are you set up to host this from your forked github.io page?

File: .gitignore (hunk 1 above)
Comment (MarlonSchultz) [reply .gitignore]: its to leave out intellij settings from the repo

**Comment by MarlonSchultz** [comment]
@lucasrangit I always build it locally on my machine via docker and present it from there. Dont have github pages set up.

File: .gitignore (hunk 1 above)
Comment (lucasrangit) [reply .gitignore]: I would drop this commit since it's only for your local machine.

**Comment by vog** [comment]
@lucasrangit I tend to approve this one, even though the `.idea` entry is only useful for IntelliJ users.

Reason: The alternative would be forcing people to keep local changes in their repos, which is never a good thing.

The situation would be different if Git had support for some kind of global `.gitignore` files (e.g. some dot-file in `$HOME`), but I'm not aware of such a Git feature.

**Comment by ligi** [comment]
merging this now - but I tend to agree with @vog regarding the .idea folder. Let's discuss this at next BHNT. But I do not think it is a blocker to merge as it is not a *big deal* - so merging this now

**Pull request merged**